modules/datetime.js

```diff
@@ -0,0 +1,13 @@
+// datetime.js
+export default function updateDateTime() {
+  const dateElement = document.querySelector('.date');
+  const timeElement = document.querySelector('.time');
+
+  const currentDate = new Date();
+
+  const formattedDate = currentDate.toDateString();
+  const formattedTime = currentDate.toLocaleTimeString();
+
+  dateElement.textContent = formattedDate;
+  timeElement.textContent = formattedTime;
+}
```

Comment:
- [ ] @efmomoh Great work adding the current date and time, however, you did it wrongly, in this milestone we are supposed to use the [Luxon](https://moment.github.io/luxon/#/install?id=es6) library. Importing it using npm and then Importing it as a module, kindly consider changing the implementation to the recommended one.
![image](https://github.com/efmomoh/my-books/assets/61408860/7a939656-e1db-420b-a3f0-42fadbbb6c3d)